mautic/mautic
# [UI] Marketplace details header and layout

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR brings life to plugin details.

- Adds a bigger header
- The header changes its color dynamically based on the plugin name
- All generated colors have enough contrast against the text on top
- It adapts its lightness depending on your theme (light or dark)
- Features a handcrafted background, using icons related to marketing

My original idea was to bring more items to this header (like icons/plugin logo) but this would require each package having their own icon (we're probably far from this, so left out for now)







| Before                                 | After
| -------------------------------------- | ---
|![image](https://github.com/user-attachments/assets/27ae6073-fd43-45e1-a157-e631938b9125)                                         | ![image](https://github.com/user-attachments/assets/a1e6f8c6-872f-4c97-89fb-d43b8661644d)


---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the admin menu
3. Open Marketplace
4. Open any plugins
